ctrlf.el

```diff
@@ -40,6 +40,19 @@
   "Non-nil means to always keep the current match vertically centered."
   :type 'boolean)
 
+(defcustom ctrlf-style-alist
+  '((literal      . (:prompt "literal"  :translator regexp-quote))
+    (regexp       . (:prompt "regexp"   :translator identity))
+    (fuzzy        . (:prompt "fuzzy"    :translator ctrlf--fuzzy-translate))
+    (fuzzy-regexp . (:prompt "fuzzy re"
+                             :translator ctrlf--fuzzy-regexp-translate)))
+  "The displayed name and regexp translator for CTRLF search styles.
+The key is the symbol of the search style, the value is a list composed of the
+displayed name and the regexp translator."
+  :type '(alist
+          :key-type symbol
+          :value-type (list string function)))
```

Comment:
I think this should be using the `plist` Custom type. Or alternatively it could use `(list (const :prompt) string (const :translator) function)`, which might be better. But I don't think `(list string function)` is correct.

Reply:
Done.